Frontend/src/components/Login.js

```diff
@@ -113,7 +82,10 @@ const Login = ({ setUser }) => {
             setLoading(false);
           }
           else {
-            validateAndSetUser(data);
+            if(isValidUser(data.user)) {
+              setUser(data.user);
```

Comment:
These should also be `data` instead of `data.user`.

Reply:
Last time I checked, the backend sends an object `{ user: Object }` so unless that's changed what I did is correct

Reply:
I think adding passport changed that. I did these edits locally and seemed to fix the login error.

Reply:
https://github.com/elinottonson/job-listing-tool/blob/main/Backend/endpoints/loginPost.js